e2e/tests/onboarding/01-onboarding.spec.js

```diff
@@ -3,22 +3,18 @@ const data = require('../../data');
 describe('Onboarding', () => {
 	before(async () => {
 		await device.launchApp({ permissions: { notifications: 'YES' }, delete: true });
-		await waitFor(element(by.id('onboarding-view')))
+		await waitFor(element(by.id('new-server-view')))
 			.toBeVisible()
 			.withTimeout(20000);
 	});
 
 	describe('Render', () => {
-		it('should have onboarding screen', async () => {
-			await expect(element(by.id('onboarding-view'))).toBeVisible();
+		it('should have onboarding screen as New Server View', async () => {
```

Comment:
Adding `as something` implies there's other ways to be rendered, which is not the case